### ISSUE TITLE
CSS Link Is still generated

### ISSUE THREAD

**Issue by Nithos** [Marco-Prontera/vite-plugin-css-injected-by-js#18]
When using the plugin the resulting HTML file generate should NOT contain the link to the non existing css chunk. However it still seems to be generated and points at nothing.

Am I missing something?

Vite: v2.8.0 and 2.9.0 tested

Config
```js
// vite.config.js
const { defineConfig } = require('vite')
import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js'


module.exports = defineConfig({
  plugins: [
    cssInjectedByJsPlugin()
  ],
  build: {
    assetsDir: '',
    rollupOptions: {
      output: {
        dir: undefined,
        manualChunks: undefined,
        file: './dist/foo.js',
      },
    },
  }
})
```

Result
```html
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Foo Example</title>
  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600" rel="stylesheet" />
  <script type="module" crossorigin src="./foo.js"></script>
  <link rel="stylesheet" href="./index.61bb7400.css"> <!-- SHOULD NOT BE HERE -->
</head>

<body>
  <div id="test-button"></div>
  <div id="example-container"></div>
  
</body>

</html>
```

**Comment by Marco-Prontera** [comment]
Hi @Nithos, I tried your configuration but in my case, the "link" element for the CSS is not generated.
Can you give me more information about it? Do you have an open project where this issue happens?

**Comment by Nithos** [comment]
Hi, not sure what happened, but I can no longer reproduce the issue on a new project or my original one. 
My apologies but thank you for reaching out.

**Comment by Marco-Prontera** [comment]
> Hi, not sure what happened, but I can no longer reproduce the issue on a new project or my original one. My apologies but thank you for reaching out.

Don't worry, thanks a lot anyway

**Comment by Marco-Prontera** [comment]
@CQCStan fixed with v1.5.1

**Comment by Nithos** [comment]
Awesome, thank you both.
